code/lib/builder-vite/input/iframe.html

```diff
@@ -14,17 +14,22 @@
       window.STORIES = '[STORIES HERE]';
       window.DOCS_OPTIONS = '[DOCS_OPTIONS HERE]';
       window.SERVER_CHANNEL_URL = '[SERVER_CHANNEL_URL HERE]';
-
+      
       // We do this so that "module && module.hot" etc. in Storybook source code
       // doesn't fail (it will simply be disabled)
       window.module = undefined;
-    </script>
+      </script>
     <!-- [HEAD HTML SNIPPET HERE] -->
   </head>
   <body>
     <!-- [BODY HTML SNIPPET HERE] -->
     <div id="storybook-root"></div>
     <div id="storybook-docs"></div>
-    <script type="module" src="/virtual:/@storybook/builder-vite/vite-app.js"></script>
+    <script type="module">
+      /* eslint-disable import/no-absolute-path, import/extensions, import/no-unresolved */
+      
+      import '/sb-preview/runtime.mjs';
```

Comment:
@IanVS I think this should really be:
```
import './sb-preview/runtime.mjs';
```

WDYT?

Reply:
In this case I think the two urls are equivalent since iframe.html is at the root of the site, right?  The relative path looks a little odd to me, personally, but I don't feel strongly so long as both work correctly, including when the site is served at a subpath (e.g. github pages).

Reply:
It's not always the root of the site